src/engraving/rendering/single/singledraw.h

```diff
@@ -130,6 +130,7 @@ class TextBase;
 class TextLine;
 class TextLineSegment;
 class TextLineBaseSegment;
+class TieSegment;
```

Comment:
It's debatable as to whether it *should* be possible to add ties to palettes.  It doesn't actually *work*, does it?  I couldn't get it to, and I'd be surprised if I could, as I doubt there is code to apply ties from palettes.

Reply:
I moved it, then removed it ... because there is an access to the object tree ... But I look like it can work in palettes, I'll return it.

I think the `single` should be considered not only for palettes, but also for other tasks, current and future ... so, it would be good in general cases to add all the elements, and where to use which to consider separately.